RunGoogleClassroom.py

```diff
@@ -36,21 +36,15 @@ def Parse_Args(filter1):
     parser.set_defaults(output=True)
     parser.set_defaults(file=True)
     args = parser.parse_args()
-    parsed = True
-    if (args.xlsx == None):
-        print("Missing required argument (they're all required)")
-        parser.print_help()
-        parsed = False
-    else:
-        filter1.Read_Args(args)    
-    return (parsed, args.assignment, args.xlsx, args.output, args.file, args.golden_source)
+    filter1.Read_Args(args)
+    return args
 
-def Open_Classroom():
+def Open_Classroom(args):
     store = file.Storage('token.json')
     creds = store.get()
     if not creds or creds.invalid:
         flow = client.flow_from_clientsecrets('credentials.json', SCOPES)
-        creds = tools.run_flow(flow, store)
+        creds = tools.run_flow(flow, store, flags=args)
```

Comment:
This was required for the run_flow() to work correctly, in case the user is running the script for the first time.